tests/input/odt/test_odt.py

```diff
@@ -179,13 +179,19 @@ def test_to_pdf_no_page():
     assert os.path.getsize(path_to_file) > 0
     assert re.match(test_utils.CACHE_FILE_PATH_PATTERN__PDF, path_to_file)
 
-    try:
-        with WandImage(filename=path_to_file) as pdf:
-            assert len(pdf.sequence) == 2
-    except PolicyError:
-        pytest.skip(
-            'You must update ImageMagic policy file to allow PDF files'
-        )
+    # TODO - G.M - 2019-06-24 - This part of check is unclear and doesn't work
+    # with last travis test. we should inspect about why later
+    # error returned is  wand.exceptions.WandRuntimeError:
+    # MagickReadImage returns false, but did raise ImageMagick exception.
+    # This can occurs when a delegate is missing, or returns EXIT_SUCCESS without generating a raster.
+
+    # try:
+    #     with WandImage(filename=path_to_file) as pdf:
+    #         assert len(pdf.sequence) == 2
+    # except PolicyError:
+    #     pytest.skip(
+    #         'You must update ImageMagic policy file to allow PDF files'
+    #     )
```

Comment:
@inkhey you can't skip an assert without explanation. The new test case is not validating that 2 pages have been generated... so ... This is not a TODO to fix the travis test: if tests are broken, first you should fix them, then make your PR for dropping 3.4 and addind 3.7

Reply:
Behavior here is clearly unclear as this test work perfectly in my own computer. It's seems really imagemagick based.
By the way, we can't consider test was working as expected before as test was skipped because of imagemagick policy, which mean we didn't do this test.

Reply:
> Behavior here is clearly unclear 

If the behavior is unclear, you first need to clarify (before to comment it). Who wrote the code?

> By the way, we can't consider test was working as expected before as test was skipped because of imagemagick policy, which mean we didn't do this test.

we didnt do this test *in a specific context*, not all the time.

Reply:
change done to do same check with pypdf.